packages/core/strapi/lib/types/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from './core';
+export * from './factories';
```

Comment:
I suppose we'll have to add something like this here `export * as factories from './factories';` otherwise it wouldn't consider the factories object, which causes an error when generating contentTypes